SUPPORTED_APIS.md

```diff
@@ -0,0 +1,82 @@
+# 🔧 Supported APIs for In-Memory Node 🔧
+
+> ⚠️ **WORK IN PROGRESS**: This list is non-comprehensive and being updated
+
```

Comment:
Just an idea: we could add a summary table on top with just the RPC method, if it's implemented and, maybe in the future, a link to a GH issue so others can collaborate on this.

Reply:
Yeah, kinda like a table of contents, I like it 😄 .

I definitely want to link to a GH Issue for each API that's not supported, but I have a follow-up in the coming days to fill out the APIs in this document and I will include that change as part of that PR.